api/app/tests/weather_models/test_process_grib.py

```diff
@@ -48,3 +48,9 @@ def test_calculate_wind_direction_from_uv():
         calculated_wind_direction = process_grib.calculate_wind_dir_from_u_v(
             test_case['u_float'], test_case['v_float'])
         assert round(calculated_wind_direction, 0) == test_case['expected_wind_direction']
+
+
+def test_convert_mps_to_kph():
```

Comment:
How about a test where precip is 0?

Reply:
Done! https://github.com/bcgov/wps/pull/3016/commits/bcc8f9a0a224276a5db01646fb2420f5ddffc529